src/seq_mv/csr_spgemm_device_symbl.in

```diff
@@ -19,23 +19,23 @@
 template HYPRE_Int
 hypre_spgemm_symbolic_rownnz
 < HYPRE_SPGEMM_BIN, SYMBL_HASH_SIZE[HYPRE_SPGEMM_BIN], T_GROUP_SIZE[HYPRE_SPGEMM_BIN], true >
-( HYPRE_Int m, HYPRE_Int *row_ind, HYPRE_Int k, HYPRE_Int n, bool need_ghash, HYPRE_Int *d_ia,
```

Comment:
I might be confused. This is a host function. Why do we need the item?

Reply:
I thought these were device functions for some reason. This is fixed now.